src/components/GameGrid.tsx

```diff
@@ -15,28 +15,27 @@ const GameGrid = ({ gameQuery }: Props) => {
   const { data, error, isLoading } = useGames(gameQuery);
   const skeletons = [1, 2, 3, 4, 5, 6, 7, 8];
```

Comment:
I know this is past its review, but consider adding a more dynamic skeleton list.

Reply:
I looked into this, and it actually seems like this is the better option. It fits the page nicely and doesn't cause any unwanted side effects.